aera-visualizer-window.cpp

```diff
@@ -265,6 +268,8 @@ bool AeraVisualizerWindow::addEvents(const string& runtimeOutputFilePath, QProgr
   regex compositeStateSimulatedPredictionRegex("^cst (\\d+): fact (\\d+) -> fact (\\d+) simulated pred fact icst \\[([ \\d]+)\\]$");
   // fact 59 icst[52][ 50 55]
   regex newInstantiatedCompositeStateRegex("^fact (\\d+) icst\\[\\d+\\]\\[([ \\d]+)\\]$");
+  // fact (1226) imdl mdl 75: 79 -> fact (1233) pred fact (1229) imdl mdl 74
+  regex newInstantiatedModelRegex("^fact \\(\\d+\\) imdl mdl \\d+: \\d+ -> fact \\((\\d+)\\) pred fact \\((\\d+)\\) imdl mdl \\d+$");
```

Comment:
I think you can remove newInstantiatedModelRegex . This is the event of making a requirement (a predicted imdl) which is different than what you are displaying with this pull request.